connect/cli/plugins/product/constants.py

```diff
@@ -93,6 +93,21 @@
     'I': 'Modified',
 }
 
+TRANSLATION_HEADERS = {
+    'A': 'Translation ID',
+    'B': 'Action',
+    'C': 'Context',
```

Comment:
I think the column name should be symmetric here. You have Context -> as an ID, but on the other hand, Owner -> is the owner's name.

Let's keep a rule like. "* ID" -> ID (Context ID, Owner ID)
Just simple name (Context, Owner) is the Name?

Reply:
Headers are now updated to follow the rule.